public/v4/apps/dynamic_minecraft.yml

```diff
@@ -0,0 +1,102 @@
+      MOTD: '$$cap_display_description'
+      ONLINE_MODE: '$$cap_online_mode'
+      RCON_PASSWORD: '$$cap_rcon_password'
+    ports:
+      - '$$cap_mc_port:25565'
+    volumes:
+      - $$cap_appname-data:/data
+    stdin_open: true
+    tty: true
+    restart: unless-stopped
+    caproverExtra:
+      notExposeAsWebApp: 'true'
+  $$cap_appname-web:
+    image: itzg/rcon:multiarch
+    environment:
+      RWA_USERNAME: $$cap_web_username
+      RWA_PASSWORD: $$cap_web_password
+      RWA_ADMIN: 'TRUE'
+      RWA_RCON_HOST: srv-captain--$$cap_appname
+      RWA_RCON_PASSWORD: $$cap_rcon_password
+    ports:
+      - 4327:4327
+    caproverExtra:
+      containerHttpPort: '4326'
+caproverOneClickApp:
+  variables:
+    - id: $$cap_appname
+      label: service name
+      defaultValue: mc
+    - id: $$java_version
+      label: Java version
+      description: 'Chose the Java version to run the server with. Look in the documentation for different versions: https://github.com/itzg/docker-minecraft-server#running-minecraft-server-on-different-java-version'
+      validRegex: /^(latest|java8|java8-jdk|java8-multiarch|java8-openj9|java8-graalvm-ce|java11|java11-jdk|java11-openj9|java17|java17-jdk|java17-openj9|java17-graalvm-ce|java17-alpine|java19)$/
+      defaultValue: java19
+    - id: $$cap_minecraft_version
+      label: Minecraft version
+      defaultValue: 1.19.3
+    - id: $$cap_forge_version
+      label: forge server version
+      description: 'Choose the version that works with your mc version , see: https://files.minecraftforge.net/'
+      defaultValue: 44.1.16
+    - id: $$server_type
+      label: Server type
+      description: 'Chose one of the server types from supported ones: https://github.com/itzg/docker-minecraft-server#server-types'
+      validRegex: /^(MAGMA|FORGE|FABRIC|QUILT|BUKKIT|SPIGOT|PAPER|FOLIA|PUFFERFISH|PURPUR|MOHIST|CATSERVER|CANYON|SPONGEVANILLA|LIMBO|CRUCIBLE|MODRINTH|FTBA|AUTO_CURSEFORGE|CURSEFORGE)$/
+      defaultValue: 'FORGE'
+    - id: $$cap_mc_port
+      label: Service port
+      description: 'Choose "25565" if you want to connect directly to the servers domain without any port specification.'
+      defaultValue: 25575
+      validRegex: /.{1,}/
+    - id: $$cap_web_username
+      label: admin portal username
+      defaultValue: 'admin'
+    - id: $$cap_web_password
+      label: admin portal password.
+      defaultValue: $$cap_gen_random_hex(8)
+    - id: $$cap_rcon_password
+      label: Rcon cli password
+      defaultValue: $$cap_gen_random_hex(8)
+    - id: $$cap_max_ram
+      label: Max RAM (GB)
+      defaultValue: 2
+      validRegex: /.{1,}/
+    - id: $$cap_display_name
+      label: server displayed name
+      defaultValue: Minecraft server
+    - id: $$cap_display_description
+      label: server displayed description
+      defaultValue: Minecraft server powerd by docker
+    - id: $$cap_online_mode
+      label: online mode
+      defaultValue: 'FALSE'
```

Comment:
set `defaultValue` to a string type value `'FALSE'` otherwise it would end up as the `false` boolean flag which is not correct.